code-challenges.test.js

```diff
@@ -14,54 +14,128 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+describe("fibonacciFinder", () => {
+      const fibLength1 = 6
+      // Expected output: [1, 1, 2, 3, 5, 8]
+      const fibLength2 = 10
+      // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+      it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+        expect(fibonacciFinder(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibonacciFinder(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+      })
+    })
+
+    // ReferenceError: fibonacciFinder is not defined
 
 // b) Create the function that makes the test pass.
+// pseudo: create a function called fibonacciFinder that takes in a number as an argument and returns an array of the argument numbers value as the array's length containing the fibonacci sequence up to that argument value's length. (so an argument of 6 should return an array that has 6 indexes)
+// declare a new varible named newArray that is an empty array, this will be what we .push() our values into and return when our function is called.
+// create a for loop within our function that iterates over our argument number as many times as its value (five times for 5, twelve times for 12, etc) by setting the loop to stop iterating at that arguments value (i < number).
+// in our for loop, create an if conditional statement that pushes a 1 into our newArray if the current index being iterated through strictly equals 0 or 1, this will guarantee we output the correct first two values in the fibonacci sequence (1, 1).
+// else, within or newArray we want to push the sum of the first index before our current index, and the second furthest index before our current index (newArray[i - 1] + newArray[i - 2]). (1 + 1 pushes 2, then 2 + 1 pushes 3, then 3 + 2 pushes 5, etc.)
+// after we close out our conditional statement, return newArray
+
+const fibonacciFinder = (number) => {
+  let newArray = []
```

Comment:
Just in case you want a challenge, can we start our new array as [ 1, 1 ]? How would that change your code?